index.html

```diff
@@ -811,6 +814,116 @@ <h3>
           </p>
         </section>
       </section>
+      <section>
+        <h3>
+          `id` member
+        </h3>
+        <p>
+          The [=manifest's=] <code><dfn data-export="" data-dfn-for=
+          "manifest" id="identity">id</dfn></code> member is a <a>string</a> that
+          represents the <dfn>identity</dfn> for the application. 
+          The [=identity=] takes the form of a URL, which is same origin as the
+          start URL.
+        </p>
+        <p>The [=identity=] is used by user agents to uniquely identify the
+        application universally. When the user agent sees a manifest with an
+        [=identity=] that does not correspond to an already-installed
+        application, it SHOULD treat that manifest as a description of a
+        distinct application, even if it is served from the same URL as that of
+        another application. When the user agent sees a manifest where
+        |manifest|["id"] [=url/equal|equals=] the [=identity=] of an
+        already-installed application, it SHOULD be used as a signal that this
+        manifest is a replacement for the already-installed application's
+        manifest, and not a distinct application, even if it is served from a
+        different URL than the one seen previously.
+        </p>
+        <p class="note">
+          The [=identity=] can be used by a service that collects lists of web
+          applications to uniquely identify applications.
+        </p>
+        <p class="note">
+          The [=identity=] is processed like a URL but it doesn't point to a
+          resource that can be navigated to, so it's not required to be
+          [=URL/within scope=].
+        </p>
+        <p>
+          To <dfn>process the `id` member</dfn>, given [=object=]
+          |json:JSON|, [=ordered map=] |manifest:ordered map|:
+        </p>
+        <ol class="algorithm">
+          <li>Set |manifest|["id"] to |manifest|["start_url"].
+          </li>
+          <li>If the type of |json|["id"] is not [=string=], return.
+          </li>
+          <li>If |json|["id"] is the empty string, return.
+          </li>
+          <li>Let |base origin| be |manifest|["start_url"]'s [=url/origin=].</li> 
+          <li>Let |id:URL| be the result of [=URL Parser|parsing=] |json|["id"] with  |base origin| as the base URL.
+          </li>
+          <li>If |id| is failure, return.
+          </li>
+          <li>If |id| is not [=same origin=] as |manifest|["start_url"], return.
+          </li>
+          <li>Otherwise, set |manifest|["id"] to |id|.</li>
+        </ol>
+        <aside class="example">
+          <p>Below table shows some example cases of the |identity| processing algorithm.</p>
+          <table class="data">
+            <tr>
+              <th>|json|["id"]</th>
+              <th>|manifest|["start_url"]</th>
+              <th>|manifest|["id"]</th>
+              <th>Explaination</th>
```

Comment:
Given the paragraph below the table, Explanation can go away again, right?

Reply:
oops, forgot to remove this, thanks for catching.